Dockerfile

```diff
@@ -161,6 +161,7 @@ RUN if [ "$CLAM_AV" = "yes" ]; then \
 # Install wheels from the build image
 RUN python3 -m venv /opt/venv
 ENV PATH="/opt/venv/bin:${PATH}"
+RUN python -m pip install --upgrade setuptools
```

Comment:
What's this for?

Reply:
Here setuptools is already installed from apt base and there setuptools version 59.6.0 which has vulnerabilities, to install the latest version(70.0.0 now) of setuptools I had to add this line

Reply:
What are the vulnerabilities?

Reply:
Actually, regardless of the answer, I would suggest _removing_ setuptools instead of upgrading. I don't think we need it in the production configuration, and if we did need it, it would be listed in `production.txt` and installed alongside all other packages.

Reply:
As docker scout shows, the vulnerability is https://github.com/advisories/GHSA-r9hx-vwmv-q579
Deletion also solves the problem, done ✅